pkgs/applications/science/logic/surelog/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, python3
+, pkg-config
+, libuuid
+, openjdk11
+}:
+
+stdenv.mkDerivation rec {
+  pname = "Surelog";
+  version = "2022.05.15";
+
+  src = fetchFromGitHub {
+    owner = "chipsalliance";
+    repo = pname;
+    rev = "15d3698ca5c7d45dd95b58c15e76131420cb001c";
+    hash = "sha256-dfje9yZ8ZR7x1EUxDUpKNcOWKYTPwPG6T4HzudV59R4=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+  ];
+
+  buildInputs = [
+    libuuid
+    openjdk11
```

Comment:
the jdk and python packages need to be in the nativeBuildInputs, as they are only needed at build time.